File: code_examples/4-Functions copy/5-Example_Sequence/JavaScript/main.js
// Copyright (c) 2020 Mr. Coxall All rights reserved
//
// Created by: Mr. Coxall
// Created on: Sep 2020
// This program calculates total from subtotal and tax

const prompt = require('prompt-sync')();

// this function calculates total from subtotal and tax
const HST = 0.13;
let tax;
let subTotal;
let total;

// get user input
subTotal = parseFloat(prompt("Enter the subtotal: $"));

// process input
tax = subTotal * HST;
total = subTotal + tax;

// output results
console.log();
console.log(`The HST is: $${tax.toFixed(2)}.`);
console.log(`The total cost is: $${total.toFixed(2)}.`);

console.log("\nDone.");

File: code_examples/4-Functions copy/5-Example_Sequence/C/main.c
// Copyright (c) 2020 Mr. Coxall All rights reserved
//
// Created by: Mr. Coxall
// Created on: Sep 2020
// This program calculates total from subtotal and tax

#include <stdio.h>
#include <math.h>

int main() {
    // this function calculates total from subtotal and tax
    const float HST = 0.13;
    float tax;
    float subTotal;
    float total;

    //input
    printf("Enter the subtotal: $");
    scanf("%f", &subTotal);

    // process
    tax = + subTotal * HST;
    total = subTotal + tax;
    
    // output
    printf("\n");
    printf("The HST is: $%.2f.\n", tax);
    printf("The total cost is: $%.2f.\n", total);

    printf("\nDone.\n");
    return 0;
}
